### PR TITLE
requestUserAccess fix

### PR DESCRIPTION
**Pull request by Wyliemaster**
the endless loop for none mods finally gone and they actually get told they're not mod now. also added elder mod message instead of the normal mod message for everyone. added comments to explain what i did

### CODE REVIEW

**Comment by Intelligent-Cat** [comment]
Actually you could just do if $permState >= 2 and you could just exit(2) than echoing and exiting

**Comment by Wyliemaster** [comment]
> Actually you could just do if $permState >= 2 and you could just exit(2) than echoing and exiting

I tried doing that before but the server response was just blank and it just stayed in an infinite loop ingame. Idk why it did that.

File: incl/mods/requestUserAccess.php [hunk 1]
```diff
@@ -18,9 +18,9 @@
 		if ($permState == 0) {
 			exit(-1); //fix infinite req loop
 		}
-		if ($permState != 1 || $permState != 0){ // if the mod badge level is higher than 2, it will still show elder mod badge
+		if ($permState >= 2){ // if the mod badge level is higher than 2, it will still show elder mod message
 			echo 2;
-			exit();
+			exit(); //exit(2) doesnt work for some reason
```
Comment (Jecket22) [inline incl/mods/requestUserAccess.php:23]: Maybe exit("2"); ?

Comment (Wyliemaster) [reply incl/mods/requestUserAccess.php:23]: ah yes, it worked, thanks, will update

**Pull request merged**